### PR TITLE
Bug 1550624 - Removing stale mixin: `limit-visible-lines`

### PR DESCRIPTION
**Pull request by gvn**
Testing steps: 

- Compare `dev-test-all` in Nightly and this PR to ensure no design regressions.

### CODE REVIEW

File: content-src/components/DiscoveryStreamComponents/Hero/_Hero.scss [hunk 1]
```diff
@@ -14,11 +14,12 @@ $card-header-in-hero-line-height: 20;
   }
 
   .excerpt {
-    @include limit-visibile-lines(3, 20, 14);
     @include dark-theme-only {
       color: $grey-10;
     }
 
+    line-height: 20px;
+    font-size: 14px;
```
Comment (Mardak) [inline content-src/components/DiscoveryStreamComponents/Hero/_Hero.scss]: Should these be using `$card-header-in-hero-line-height` ? The variables at the top seem to be unused.

File: content-src/components/DiscoveryStreamComponents/Hero/_Hero.scss [hunk 2]
```diff
@@ -268,7 +271,8 @@ $card-header-in-hero-line-height: 20;
             color: $white;
           }
 
-          @include limit-visibile-lines(3, 20, 14);
+          line-height: 20px;
+          font-size: 14px;
```
Comment (Mardak) [inline content-src/components/DiscoveryStreamComponents/Hero/_Hero.scss]: These also seem to match up with `$card-header-in-hero-line-height` but not sure if they should use the existing variables ?

**Comment by gvn** [comment]
I guess we should put this on the backburner for now given our decision to roll back dynamic line clamping.

**Comment by Mardak** [comment]
This was backlogged

**Pull request closed without merging**